master/internal/rm/kubernetesrm/spec.go

```diff
@@ -480,6 +481,7 @@ func (p *pod) createPodSpec(scheduler string) error {
 
 	p.pod = p.configurePodSpec(
 		volumes, initContainer, container, sidecars, (*k8sV1.Pod)(env.PodSpec()), scheduler)
+	p.insertLog(time.Now().UTC(), fmt.Sprintf("Pod Spec Configured:\n%v\n", p.pod.Spec))
```

Comment:
is the message obnoxiously large? Like do you think it could be annoying for some users?

Maybe we could add the log at a debug level?

Reply:
I'm afraid that if we log it at the debug level it might not be useful enough in a lot of situations. I personally don't think it's too obnoxious, but what do you think? (I'll add a screenshot)

Reply:
![image](https://github.com/determined-ai/determined/assets/31023784/b22f1e10-1be0-4f1a-bbad-386b41f5cdfa)